test/t8n/t8n.cpp

```diff
@@ -106,9 +109,8 @@ int main(int argc, const char* argv[])
                     auto tx = test::from_json<state::Transaction>(j_txs[i]);
                     tx.chain_id = chain_id;
 
-                    auto res = state::transition(state, block, tx, rev, vm);
-
                     const auto computed_tx_hash = keccak256(rlp::encode(tx));
+                    const auto computed_tx_hash_str = hex0x(computed_tx_hash);
```

Comment:
This is a refactoring, perhaps merge it separately?

Reply:
How about separated commit?